wandb/integration/keras/keras.py

```diff
@@ -215,34 +215,37 @@ class WandbCallback(keras.callbacks.Callback):
     """WandbCallback automatically integrates keras with wandb.
 
     Example:
-        ```
-        model.fit(X_train, y_train,  validation_data=(X_test, y_test),
-            callbacks=[WandbCallback()])
+        ```python
+        model.fit(X_train,
+            y_train,
+            validation_data=(X_test, y_test),
+            callbacks=[WandbCallback()]
+        )
         ```
 
     WandbCallback will automatically log history data from any
```

Comment:
I would suggest adding backticks to all instances of `WandbCallback` in the docstring, plus any other place where we we're using variable names/values. For example, in `monitor`: `auto`, `min`, and `max` should all be code-formatted, `model.fit` in the description of `training_data`, etc.

Reply:
Done!